distribution/feature-packs/server-feature-pack/src/main/resources/modules/system/layers/keycloak/org/keycloak/keycloak-server-subsystem/dependencies/main/module.xml

```diff
@@ -23,6 +23,7 @@
 
     <dependencies>
         <module name="org.keycloak.keycloak-services" export="true"/>
+        <module name="org.keycloak.keycloak-wildfly-extensions" export="true"/>
```

Comment:
This shouldn't be needed as platform provider is used within services only, which has a dependency on keycloak-wildfly-extensions

Reply:
done; had to add `export="true"` to `services` → `wildfy-extensions` dep to make the latter visible to the WAR